src/main.rs

```diff
@@ -66,6 +68,14 @@ impl BoltArgs {
     }
 }
 
+#[derive(clap::Parser, Debug)]
+#[clap(trailing_var_arg(true))]
+struct CleanArgs {
+    /// Override the PGO profile directory.
+    #[clap(long)]
+    override_pgo_directory: Option<PathBuf>,
```

Comment:
Let's call it `profiles_dir` (in all commands), to keep it shorter, and make it explicit that we're talking about the profiles.